src/components/Navigation/ConversationsList/ConversationsListItem.vue

```diff
@@ -0,0 +1,188 @@
+<!--
+  - @copyright Copyright (c) 2019 Joas Schilling <coding@schilljs.com>
+  -
+  - @author Joas Schilling <coding@schilljs.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<AppContentListItem
+		:title="item.displayName"
+		@click.prevent.exact="joinConversation(item.token)">
+		<ConversationsIcon
+			slot="icon"
+			:item="item" />
+		<template slot="subtitle">
+			{{ item.lastMessage.message }}
+		</template>
+		<AppNavigationCounter v-if="item.unreadMessages"
+			slot="counter"
+			:highlighted="true">
+			{{ item.unreadMessages }}
+		</AppNavigationCounter>
+		<template slot="actions">
+			<ActionButton v-if="canFavorite"
+				:icon="iconFavorite"
+				@click.prevent.exact="toggleFavoriteConversation">
+				{{ labelFavorite }}
+			</ActionButton>
+
+			<!-- FIXME Should be a real separator -->
+			<ActionText>
+				FIXME I'm a separator
+			</ActionText>
+
+			<ActionText>
+				{{ t('spreed', 'Chat notifications') }}
+			</ActionText>
+			<ActionButton
+				icon="icon-sound"
+				@click.prevent.exact="setNotificationLevel(1)">
+				{{ t('spreed', 'All messages') }}
+			</ActionButton>
+			<ActionButton
+				icon="icon-user"
+				@click.prevent.exact="setNotificationLevel(2)">
+				{{ t('spreed', '@-mentions only') }}
+			</ActionButton>
+			<ActionButton
+				icon="icon-sound-off"
+				@click.prevent.exact="setNotificationLevel(3)">
+				{{ t('spreed', 'Off') }}
+			</ActionButton>
```

Comment:
Can not set the active class, ActionRadio however makes us loose the icon

Reply:
Old template code
https://github.com/nextcloud/spreed/blob/e94805d/js/views/templates/roomlistview.handlebars#L48-L65

Reply:
@ma12-co I'm very unsure about this one. And idea what would be the best way to allow this?

Reply:
@nickvergessen  we've been discussing this yesterday at the design meeting and I'm going to add this capability to the actions component.